src/backend/cdb/cdbpath.c

```diff
@@ -81,7 +81,9 @@ cdbpath_cost_motion(PlannerInfo *root, CdbMotionPath *motionpath)
  *    If no motion is needed, the caller's subpath is returned unchanged.
  *    Else if require_existing_order is true, NULL is returned if the
  *      motion would not preserve an ordering at least as strong as the
- *      specified ordering; also NULL is returned if pathkeys is NIL
+ *      specified ordering or if path has parameterization relids, 
+ *      which generally may indicate we can't add motion; 
```

Comment:
Extra spaces were appended to two lines above

Reply:
Resolved